PIL/Image.py

```diff
@@ -1122,6 +1186,7 @@ def filter(self, filter):
         return merge(self.mode, ims)
 
     def getbands(self):
+        # type: () -> Tuple[Any]
```

Comment:
Instead of `Tuple[Any]`, couldn't this be `Tuple[str, ...]`, or...I'm so confused by  `ImageMode.py`, and why are there so many `global` variables? It should be far easier to narrow down what this method should return as a type.